src/pages/settings/Payments/TransferBalancePage.js

```diff
@@ -119,12 +118,21 @@ class TransferBalancePage extends React.Component {
      * @param {Number} transferAmount
      * @param {Object} selectedAccount
      */
-    saveTransferAmountAndBalance(transferAmount, selectedAccount) {
+    saveTransferAmountAndStartTransfer(transferAmount, selectedAccount) {
         PaymentMethods.saveWalletTransferAmount(transferAmount);
         PaymentMethods.transferWalletBalance(selectedAccount);
```

Comment:
NAB but I still do not like that it's difficult to tell which of these is calling the API and which is not.

Reply:
I agree with your point here but I will wait for a clear direction from you on handling this. Maybe we can leave this for refactoring later on.

Reply:
Yes I'm sorry I don't know how to improve it right now and it might not be a problem so "not a blocker"